### PR TITLE
Dropdown-component-issue

### PR DESCRIPTION
**Pull request by Navin1196**
Dropdown Issues fixed.
1.Access to open and close the dropdown has been applied to both the icon and the entire div.
2.The spacing issue within the group dropdown in the user edit modal has been resolved.
3.MAC responsive design fixed.

image
image

### CODE REVIEW

File: src/components/reusable/dropdown/index.tsx [hunk 1]
```diff
@@ -85,7 +91,13 @@ const Dropdown = ({
   };
 
   return (
-    <div className="relative w-full">
+    <div
+      className={`relative w-full ${wrapClassName}`}
```
Comment (chandran-kannan) [inline src/components/reusable/dropdown/index.tsx]: wrapClassName || ''

File: src/components/reusable/dropdown/index.tsx [hunk 2]
```diff
@@ -85,7 +91,13 @@ const Dropdown = ({
...
+      onClick={(e) => {
```
Comment (chandran-kannan) [inline src/components/reusable/dropdown/index.tsx]: `e` doesn't sounds good, name it `event` for better readability

**Review by chandran-kannan — CHANGES_REQUESTED** [review]

File: src/components/layout/sidenav/index.tsx [hunk 3]
```diff
@@ -21,7 +21,7 @@ const SideNav = () => {
       >
         <OmnexIcon />
       </div>
-      <div className="flex flex-col justify-center items-center gap-[10px] overflow-y-auto">
```
Comment (chandran-kannan) [inline src/components/layout/sidenav/index.tsx]: why overflow-y-auto removed?

File: src/components/layout/index.tsx [hunk 4]
```diff
@@ -22,12 +22,12 @@ export function Layout() {
         })} flex h-full w-full overflow-hidden py-[14px] px-[14px]`}
       >
         <SideNav />
-        <div className="px-[30px] w-full h-auto overflow-y-auto flex flex-col">
```
Comment (chandran-kannan) [inline src/components/layout/index.tsx:25]: why overflow-y-auto removed?

**Review by chandran-kannan — CHANGES_REQUESTED** [review]

File: src/components/layout/sidenav/index.tsx (hunk 3 above)
Comment (Navin1196) [reply src/components/layout/sidenav/index.tsx]: As a result of utilizing 'overflow-y-auto', an unintended vertical scrolling issue is occurring specifically on Mac screens.

File: src/components/layout/index.tsx (hunk 4 above)
Comment (Navin1196) [reply src/components/layout/index.tsx:25]: As a result of utilizing 'overflow-y-auto', an unintended vertical scrolling issue is occurring specifically on Mac screens.

File: src/pages/users/EditUser.tsx [hunk 5]
```diff
@@ -156,7 +157,7 @@ const EditUser = ({
             variant="primary"
             label="Submit"
             onClick={updateUser}
-            className="py-[8px] px-[24px] rounded-[16px] font-normal text-[16px]"
+            className="py-[8px] px-[24px] -z-50 rounded-[16px] font-normal text-[16px]"
```
Comment (chandran-kannan) [inline src/pages/users/EditUser.tsx]: remove z index

**Review by chandran-kannan — CHANGES_REQUESTED** [review]

File: src/components/reusable/dropdown/index.tsx (hunk 1 above)
Comment (Navin1196) [reply src/components/reusable/dropdown/index.tsx]: Fixed it.

File: src/components/reusable/dropdown/index.tsx (hunk 2 above)
Comment (Navin1196) [reply src/components/reusable/dropdown/index.tsx]: Fixed it.

File: src/pages/users/EditUser.tsx (hunk 5 above)
Comment (Navin1196) [reply src/pages/users/EditUser.tsx]: Fixed it.

**Review by chandran-kannan — APPROVED** [review]

**Pull request merged**